.gitignore

```diff
@@ -20,4 +20,4 @@ erl_crash.dump
 *.swo
 *~
 
-serum
+/serum
```

Comment:
Unlike the intention, it matched so broadly. So I changed to match only the top.